quickstart/docker-compose.yml

```diff
@@ -4,106 +4,86 @@ version: "3.9"
+  opensearch:
+    image: public.ecr.aws/opensearchproject/opensearch:2.6.0
+    hostname: opensearch
+    container_name: opensearch
     environment:
-      - node.name=elasticsearch
-      - cluster.name=es-docker-cluster
-      - cluster.initial_master_nodes=elasticsearch
-      - bootstrap.memory_lock=true
-      - xpack.security.enabled=false
-      - "ES_JAVA_OPTS=-Xms512m -Xmx512m"
+      - cluster.name=opensearch-cluster # Name the cluster
+      - node.name=opensearch # Name the node that will run in this container
+      - discovery.seed_hosts=opensearch # Nodes to look for when discovering the cluster
+      - cluster.initial_cluster_manager_nodes=opensearch # Nodes eligibile to serve as cluster manager
+      - bootstrap.memory_lock=true # Disable JVM heap memory swapping
+      - "OPENSEARCH_JAVA_OPTS=-Xms512m -Xmx512m" # Set min and max JVM heap sizes to at least 50% of system RAM
+      - "DISABLE_INSTALL_DEMO_CONFIG=true" # Prevents execution of bundled demo script which installs demo certificates and security configurations to OpenSearch
+      - "DISABLE_SECURITY_PLUGIN=true" # Disables security plugin
     ulimits:
       memlock:
         soft: -1
         hard: -1
+      nofile:
+        soft: 65536 # Maximum number of open files for the opensearch user - set to at least 65536
+        hard: 65536
     volumes:
-      - data:/usr/share/elasticsearch/data
+      - data:/usr/share/opensearch/data
     ports:
       - 9200:9200
     networks:
-      elastic:
-        ipv4_address: 172.21.0.2
+      - opensearch-net
 
-  kibana:
-    image: docker.elastic.co/kibana/kibana:7.10.2
-    container_name: kibana
+  dashboards:
+    image: public.ecr.aws/opensearchproject/opensearch-dashboards:2.6.0
+    container_name: dashboards
     ports:
       - 5601:5601
     environment:
-      ELASTICSEARCH_URL: http://elasticsearch:9200
-      ELASTICSEARCH_HOSTS: http://elasticsearch:9200
+      - 'OPENSEARCH_HOSTS=["http://opensearch:9200"]'
+      - "DISABLE_SECURITY_DASHBOARDS_PLUGIN=true" # disables security dashboards plugin in OpenSearch Dashboards
     networks:
-      elastic:
-        ipv4_address: 172.21.0.3
+      - opensearch-net
     depends_on:
-      - elasticsearch
-
-  zookeeper:
-    image: wurstmeister/zookeeper
-    hostname: zookeeper
-    container_name: zookeeper
-    ports:
-      - 2181:2181
-    networks:
-      elastic:
-        ipv4_address: 172.21.0.4
+      - opensearch
 
   kafka:
-    image: wurstmeister/kafka
+    image: bitnami/kafka:latest
     container_name: kafka
     ports:
       - 9092:9092
     hostname: kafka
     networks:
-      elastic:
-        ipv4_address: 172.21.0.5
+      - opensearch-net
     environment:
-      KAFKA_ADVERTISED_HOST_NAME: kafka
-      KAFKA_ADVERTISED_LISTENERS: PLAINTEXT://172.21.0.5:9092
-      KAFKA_CREATE_TOPICS: "consumer:1:1,producer:1:1,producer_error:1:1,pseudonyms:1:1,sre:1:1"
-      KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
+      - KAFKA_ENABLE_KRAFT=yes
```

Comment:
this setting seems to require another Environment Variable `KAFKA_CFG_NODE_ID`, when I'm trying to run this docker-compose deployment with a fresh bitnami/kafka:latest I'm getting the following error and Kafka will exit:

![image](https://user-images.githubusercontent.com/14598229/233972219-f5a06384-2754-417b-9387-c739497bb1aa.png)

Reply:
fixed